adding 5 youtube demos to different parts of the docs (#1138)
wandb/docs: content/guides/hosting/iam/scim.md

weight: 4
---

{{% alert %}}

> Prefer a video? Watch [this demo](https://www.youtube.com/watch?v=Nw3QBqV0I-o) (12 min)

Reviewer: ```suggestion
Watch a [video demonstrating SCIM in action](https://www.youtube.com/watch?v=Nw3QBqV0I-o) (12 min)
```
